Dockerfile

```diff
@@ -1,40 +1,65 @@
-ARG GOLANG_BUILDER=golang:1.13
+ARG GOLANG_BUILDER=golang:1.14
 ARG OPERATOR_BASE_IMAGE=gcr.io/distroless/static:nonroot
 
 # Build the manager binary
 FROM ${GOLANG_BUILDER} AS builder
 
-ARG GO_BUILD_EXTRA_ARGS
+#Arguments required by OSBS build system
+ARG CACHITO_ENV_FILE=/remote-source/cachito.env
+
+ARG REMOTE_SOURCE=.
+ARG REMOTE_SOURCE_DIR=/remote-source
+ARG REMOTE_SOURCE_SUBDIR=
+ARG DEST_ROOT=/dest-root
+
+COPY $REMOTE_SOURCE $REMOTE_SOURCE_DIR
+WORKDIR ${REMOTE_SOURCE_DIR}/${REMOTE_SOURCE_SUBDIR}
+
+RUN go version
+RUN go env
+RUN if [ -f $CACHITO_ENV_FILE ] ; then echo $CACHITO_ENV_FILE; cat $CACHITO_ENV_FILE ; fi
+RUN if [ -f $CACHITO_ENV_FILE ] ; then source $CACHITO_ENV_FILE ; fi && go env
+
+RUN mkdir -p ${DEST_ROOT}/usr/local/bin/
 
-WORKDIR /workspace
-# Copy the Go Modules manifests
-COPY go.mod go.mod
-COPY go.sum go.sum
 # cache deps before building and copying source so that we don't need to re-download as much
 # and so that source changes don't invalidate our downloaded layer
-RUN go mod download
+RUN if [ ! -f $CACHITO_ENV_FILE ]; then go mod download ; fi
 
-# Copy the go source
-COPY main.go main.go
-COPY api/ api/
-COPY pkg/ pkg/
-COPY controllers/ controllers/
-COPY templates/ templates/
-RUN mkdir -p /usr/share/osp-director-operator/templates && mkdir -p /cmd/
+#RUN mkdir -p /usr/share/osp-director-operator/templates && mkdir -p /cmd/
 
 # Build manager
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build ${GO_BUILD_EXTRA_ARGS} -a -o manager main.go
+RUN if [ -f $CACHITO_ENV_FILE ] ; then source $CACHITO_ENV_FILE ; fi ; CGO_ENABLED=0  GO111MODULE=on ${GO_BUILD_EXTRA_VARS} go build ${GO_BUILD_EXTRA_ARGS} -v -a -o ${DEST_ROOT}/manager main.go
+
+RUN cp -r templates ${DEST_ROOT}/templates
+
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
 FROM ${OPERATOR_BASE_IMAGE}
 
+ARG DEST_ROOT=/dest-root
+ARG USER_ID=nonroot:nonroot
+
+LABEL   com.redhat.component="osp-director-operator-container" \
+        name="osp-director-operator" \
+        version="1.0" \
+        summary="OSP Director Operator" \
+        io.k8s.name="nova-operator" \
```

Comment:
copy paste pasta mixup, name change didn't happen here.

Reply:
?

Reply:
ok got it s/nova/osp-director